pkg/sqlqueries/mto_milestone_solution_link.go

```diff
@@ -11,17 +11,23 @@ var mtoMilestoneGetMilestoneSolutionLinksByMilestoneIDSQL string
 //go:embed SQL/mto/milestone_solution_link/merge_solutions_to_milestone.sql
 var mtoMilestoneSolutionLinkMergeSolutionsToMilestonesSQL string
 
+//go:embed SQL/mto/milestone_solution_link/link_milestones_to_solution.sql
+var mtoSolutionLinkMilestonesToSolutionSQL string
+
 type mtoMilestoneSolutionLinkScripts struct {
 	// creates a link between a milestone and a solution and returns the link ID
 	Create string
 	// returns all milestone-solution links for a given milestone by ID
 	GetByMilestoneID string
 	// Takes an array of solution_ids and a milestone_id and performs a merge to ensure only these solutions are listed to that milestone
 	MergeSolutionsToMilestones string
+	// Takes an array of milestone_ids and a solution_id and performs a merge to ensure only these milestones are listed to that solution
+	LinkMilestonesToSolution string
```

Comment:
nit: maybe these should use the same verb as it does for the Milestone centric query? (It might be clearer when we come back to it later that they are doing similar things

Reply:
This was a conscious decision in that I feel the previous naming convention has some issues. I resisted the urge to rename old methods, perhaps as a 1 point tech lift later.